sdk/go/examples/auth/main.go

```diff
@@ -0,0 +1,25 @@
+/*
+ * This example is part of the Modus project, licensed under the Apache License 2.0.
+ * You may modify and use this example in accordance with the license.
+ * See the LICENSE file that accompanied this code for further details.
+ */
+
+package main
+
+import (
+	"github.com/hypermodeinc/modus/sdk/go/pkg/auth"
+)
+
```

Comment:
See comments about adding comments from the AssemblyScript example.  Same should be done here.